src/Explorer/Tree/ResourceTreeAdapter.tsx

```diff
@@ -351,6 +357,80 @@ export class ResourceTreeAdapter implements ReactAdapter {
     };
   }
 
+  private buildSchemaNode(collection: ViewModels.Collection): TreeNode {
+    if (collection.analyticalStorageTtl() == undefined) {
+      return null;
```

Comment:
Use `undefined` when possible. We are trying to remove all `null`'s from the code (because it's redundant with `undefined` and we have to check for both. Steve will turn on a lint rule for it).